pkg/catalog/configmeta.go

```diff
@@ -189,6 +189,9 @@ func genConfigConnections(
 	kdcm := make(map[string][]map[string]map[string]string)
 	for _, connectedCmName := range cr.Spec.Connections.ConfigMaps {
 		cm, err := observer.GetConfigMap(cr.Namespace, connectedCmName)
+		if err != nil {
```

Comment:
So e.g. going by my comment about error handling...

This seems good if the error is "not found".

However if the error is NOT "not found", but rather some other problem talking to the K8s API, we can return an error here. Let's see what would happen then...

* In the caller clusterBaseConfig we can return the error if any of the connection-generation functions return an error.
* In the caller of THAT, ConfigmetaGenerator, we can return the error if clusterBaseConfig returns an error.
* The caller of THAT, syncCluster, already handles those errors and exits the handler. We've already bumped SpecGenerationToProcess by that point, so we should be set to try again on the next handler (but please double-check that logic).